tests/ui/impl-trait/issues/issue-79099.rs

```diff
@@ -1,7 +1,7 @@
 struct Bug {
     V1: [(); {
         let f: impl core::future::Future<Output = u8> = async { 1 };
-        //~^ `impl Trait` only allowed in function and inherent method argument and return types
+        //~^ `impl Trait` is not allowed in variable bindings
```

Comment:
`in variable bindings` is fine but `in the type of variable bindings` imo, not super important